### ISSUE TITLE
Feature Request: Dark theme

### ISSUE THREAD

**Issue by NightMean** [linkwarden/browser-extension#49]
Hi, thanks for making this extension! :)

As linkwarden itself supports dark theme, it would be great addition if extension would support it as well. 
What I would like is either toggle in the config or that the extension would respect the Linkwarden/System settings.

**Comment by crkos** [comment]
Hello thanks for the suggestion, of course, since we are using shadcn ui, This will be easy to implement, It will probably be shipped on the next update, i'm doing some automating sync when bookmarking if wanted on the options of the extension, So just wait some it will be shipped along with it!

**Comment by NightMean** [comment]
Perfect, thanks a lot ! I'll happily wait :)

**Comment by crkos** [comment]
Dark theme was implemented!, if you wish to install right now i would advise to check "dev"  branch, if not wait until the update on stores, I'm working on some updates for the extension that would resolve this, #45 and #36

**Comment by NightMean** [comment]
Whoa, thanks for the quick implementation! :) Apologize for late reply, I've just tested that and it works as expect. Can't wait for the update. 
Just one minor thing, the initial config screen is still white without taking in account the system dark theme. Not sure if this is how you wanted it or if it was forgotten by mistake. 

![image](https://github.com/linkwarden/browser-extension/assets/5726996/045df526-5e76-4a44-94df-57bc518b2d21)

**Comment by crkos** [comment]
Yeah i think that is due to those being completly different pages, so i didnt remember that while doing it, that will be solved on the next update

**Comment by arnavpraneet** [comment]
Hi, unrelated to that but I saw you mentioned right click to save in context menu in this issue, are you still working on the update/is it done already? I was trying my hand at it but I am very new to extensions and all and haven't had much luck. Thanks!

**Comment by crkos** [comment]
Hello, you should be able to right click on any web page and be able to add the link

**Comment by arnavpraneet** [comment]
Does this work on the latest version on Firefox or Chrome? I tried with both from the extension store, didn't work for me. Nothing in the context menu.

**Comment by crkos** [comment]
It should appear like this, It supported on  both:
![contextmenu](https://github.com/linkwarden/browser-extension/assets/92697104/cb13ccb7-d963-4637-97b3-8a0cef30c2fa)

**Comment by arnavpraneet** [comment]
Still no dice, not working for either. Anything I can check to see what's wrong? Sorry a little new to this stuff

**Comment by crkos** [comment]
I dont really know what could be wrong, It is there, I mean unless someone has the same problem, I cant really know for sure what could be the problem

**Comment by crkos** [comment]
Well if you want send a ss of the content menu when you right click on a page.

**Comment by arnavpraneet** [comment]
Understandable, I will check it on other systems to see if its affecting my system only, and wait for if anyone has the same problem. Thanks for your time!
I will attach an SS of the context menu when I right click on a page as well.
![image](https://github.com/linkwarden/browser-extension/assets/30770259/2cc6b22c-fb7b-4dfe-93c9-d1a917ffedcd)
![image](https://github.com/linkwarden/browser-extension/assets/30770259/112665f8-a8b0-4574-adaf-4ee6e7de48d2)
first screenshot when right clicking on a link, second when you right click on the page without selecting any link.

**Comment by crkos** [comment]
If this goes on please just make a new issue, because this thread is about something else.

**Comment by daniel31x13** [comment]
@arnavpraneet could you confirm if you’re on the latest version of the extension?

**Comment by arnavpraneet** [comment]
yes @daniel31x13, shall I make a new issue now?
